lib/sda/node.go

```diff
@@ -135,6 +181,19 @@ func (n *Node) EntityList() *EntityList {
 	return n.Tree().EntityList
 }
 
+// List returns the list of TreeNodes cached in the node (creating it if necessary)
+func (n *Node) List() []*TreeNode {
+	if n.treeNodeList == nil {
+		n.treeNodeList = n.Tree().List()
+	}
```

Comment:
As @ineiti said, we might not want to do that for testing reason because when you have 60,000 nodes, it's not something you want to do each time you create a `Node`